tests/h/search/query_test.py

```diff
@@ -108,6 +109,24 @@ def search(self, pyramid_request):
 
 
 class TestUserFilter(object):
+    def test_filters_annotations_by_user(self, search, Annotation):
+        Annotation(userid="acct:foo@auth2", shared=True)
+        expected_ids = [Annotation(userid="acct:bar@auth2", shared=True).id]
+
+        result = search.run({'user': "bar"})
+
+        assert set(result.annotation_ids) == set(expected_ids)
+
+    def test_filters_annotations_by_multiple_users(self, search, Annotation):
+        Annotation(userid="acct:foo@auth2", shared=True)
+        expected_ids = [Annotation(userid="acct:bar@auth2", shared=True).id,
+                      Annotation(userid="acct:baz@auth2", shared=True).id]
+        params = webob.multidict.MultiDict()
+        params.add("user", "bar")
+        params.add("user", "baz")
+        result = search.run(params)
```

Comment:
If I'm being super nit-picky we usually put a blank line between the "arrange", "act" and "assert" phases of a test, so before the `search.run` line here.